lldb/tools/lldb-dap/Handler/RequestHandler.h

```diff
@@ -47,7 +48,7 @@ class BaseRequestHandler {
 
   virtual void operator()(const protocol::Request &request) const = 0;
 
-  virtual llvm::StringMap<bool> GetCapabilities() const { return {}; }
+  virtual std::set<AdapterFeature> GetSupportedFeatures() const { return {}; }
```

Comment:
Should this return an `llvm::SmallSet<AdapterFeature,1>?`? And then merge them into a `llvm::DenseSet`?

Reply:
Done.